src/client/Microsoft.Identity.Client/TokenCache.ITokenCacheInternal.cs

```diff
@@ -255,9 +255,9 @@ public sealed partial class TokenCache : ITokenCacheInternal
             }
         }
 
-        private DateTimeOffset CalculateSuggestedCacheExpiry()
+        private DateTimeOffset CalculateSuggestedCacheExpiry(string partitionKey)
```

Comment:
Same obs as before with HasTokens() - this is used for the distributed cache only. MSAL should not need this. And for the distributed cache, the logic should be to look at all tokens. 

This is a good candidate to move to IAccessor interface maybe.

Reply:
Updated.